nltk/twitter/twitter_demo.py

```diff
@@ -0,0 +1,231 @@
+# -*- coding: utf-8 -*-
+# Natural Language Toolkit: Twitter client
+#
+# Copyright (C) 2001-2015 NLTK Project
+# Author: Ewan Klein <ewan@inf.ed.ac.uk>
+# URL: <http://nltk.org/>
+# For license information, see LICENSE.TXT
+
+"""
+Examples to demo the :py:mod:`twitterclient` code.
```

Comment:
I see. The second best place for such demos with docs would be a howto file in `test/twitter.doctest` to appear at http://www.nltk.org/howto/